composer.json

```diff
@@ -14,11 +14,12 @@
     ],
     "license": "GPL-2.0-or-later",
     "require": {
-        "php": ">=7.0.8",
+        "php": ">=7.1",
```

Comment:
do we have to bump the PHP version? I would like to keep it if we don't have to drop it.

Reply:
Yep we do... see https://github.com/slevomat/coding-standard/blob/1ace9db4f34d34692f5342f6e51fe7145472cb7b/composer.json#L18